src/section-factoring-out-the-common-factor.ptx

```diff
@@ -299,45 +299,45 @@
 
     <subsection>
         <title>More Examples of Factoring out the Common Factor</title>
-
-        <p>The previous subsection did not have examples that cover every nuance with factoring out the greatest common factor. Here are a few more factoring examples that attempt to do so.</p>
-
+        <p></p>
```

Comment:
Is this empty p intentional, or a remnant from editing? We should never make empty p's. If it was intentional to insert some space or something like that, there is a different issue to address.